conf/sample.ini

```
@@ -292,6 +292,9 @@
 # $ROOT_PATH is server.root_url without the protocol.
 ;content_security_policy_template = """script-src 'self' 'unsafe-eval' 'unsafe-inline' 'strict-dynamic' $NONCE;object-src 'none';font-src 'self';style-src 'self' 'unsafe-inline' blob:;img-src * data:;base-uri 'self';connect-src 'self' grafana.com ws://$ROOT_PATH wss://$ROOT_PATH;manifest-src 'self';media-src 'none';form-action 'self';"""
 
+# Controls if old angular plugins are supported or not. This will be disabled by default in Grafana v9.
+angular_support_enabled = true
```

```suggestion
;angular_support_enabled = true
```
Based on the rest of `sample.ini` I think this needs to be commented out.

---

ah, good catch!